Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -53,8 +53,11 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed?
         return unless MacOS::Xcode.outdated?
 
-        # CI images are going to end up outdated so don't complain.
-        return if ENV["TRAVIS"] || ENV["CIRCLECI"]
+        # Travis CI images are going to end up outdated so don't complain when
+        # `brew test-bot` runs `brew doctor` in the CI for the Homebrew/brew
+        # repository. This only needs to support whatever CI provider
+        # Homebrew/brew is currently using.
+        return if ENV["TRAVIS"]
```

Comment:
Ah, I didn't even realize that this had nothing to do with Linuxbrew. I just saw "CIRCLECI" and assumed one of us put it there. But it's actually in extend/os/mac, so there's no action required on our end.